runtime/lib/app-runtime.js

```diff
@@ -1134,11 +1134,10 @@ AppRuntime.prototype.onLoggedIn = function () {
     this.startDaemonApps(),
     this.setStartupFlag(),
     this.initiate()
-      .then(deferred, err => {
-        logger.error('Unexpected error on runtime.initiate', err.stack)
-        return deferred()
-      })
-  ]).then(onDone, err => {
+  ]).then(deferred, err => {
+    logger.error('Unexpected error on runtime.initiate', err.stack)
```

Comment:
log `Unexpected error on runtime.initiate` should be changed.

Reply:
Updated, thanks :)